locale/tr/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Çoğu kullanıcı için önerilir
   changelog: Değişiklikler
   api: API Belgeleri
-  version-schedule-prompt: Ya da
-  version-schedule-prompt-link-text: Uzun Süreli Destek (LTS) takvimine bakın
+  version-schedule-prompt: Desteklenen sürümler hakkında daha fazla bilgi için
+  version-schedule-prompt-link-text: sürüm planına bakıniz
```

Comment:
The exact translation for this line us “Look at the version release schedule”  If that’s the case this should be “sürüm yayınlanma planına bakınız". If the translation is for “Release schedule” only: it should be “Sürüm yayınlanma planı"

Reply:
```suggestion
  version-schedule-prompt-link-text: sürüm yayınlanma planına bakınız
```